src/content/platform-integration/web/faq.md

```diff
@@ -35,6 +35,11 @@ consider separating your primary application experience—created in Flutter—f
 your landing page, marketing content, and help content—created using
 search-engine optimized HTML.
 
+Having said that, as we mention in our [roadmap][], we plan to investigate
+search engine indexability of Flutter Web. To that end we've built a small
+website containing [Hawaii-themed space stories][space_hawaii], which we
+would like search engines to pick up and index.
```

Comment:
**suggestion:** Would recommend tightening somewhat as the quantity of "we"s is a bit much.

```suggestion
That said, as mentioned in the [roadmap][], the Flutter team plans to investigate
search engine indexability of Flutter Web.
To that end, we built a small website containing [Hawaii-themed space stories][space_hawaii].
We intend for search engines to find and index this site.
```

Reply:
Further feedback on Tony's feedback:

- It's "Flutter web" NOT "Flutter Web"
- "To that end, we built a small website containing [Hawaii-themed space stories][space_hawaii].
We intend for search engines to find and index this site."  =>
  To that end, we built a small website containing [Hawaii-themed space stories][space_hawaii],
 hoping that search engines find and index this site."

Reply:
Done. Thank you!